web-admin/src/components/home/DashboardList.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import Axios from "axios";
+  import {
+    useAdminServiceGetProject,
+    V1GetProjectResponse,
+  } from "../../client";
+
+  export let organization: string;
+  export let project: string;
+
+  let dashboards: string[];
+
+  $: proj = useAdminServiceGetProject(organization, project);
+  $: if ($proj.isSuccess && $proj.data?.productionDeployment) {
+    getDashboardsForProject($proj.data);
+  }
+
+  async function getDashboardsForProject(projectData: V1GetProjectResponse) {
+    // Hack: in development, the runtime host is actually on port 8081
+    const runtimeHost = projectData.productionDeployment.runtimeHost.replace(
+      "localhost:9091",
+      "localhost:8081"
+    );
+
+    const axios = Axios.create({
+      baseURL: runtimeHost,
+      headers: {
+        Authorization: `Bearer ${projectData.jwt}`,
+      },
+    });
+
+    const { data: filesData } = await axios.get(
+      `/v1/instances/${projectData.productionDeployment.runtimeInstanceId}/files`
+    );
+
+    // Filter for dashboard files & extract dashboard names
+    dashboards = filesData.paths
+      ?.filter((path) => path.includes("dashboards/"))
+      // Remove "gitkeep" files
+      .filter((path) => !path.includes(".gitkeep"))
+      // Remove "dashboards/" prefix and ".yaml" suffix
+      .map((path) => path.replace("/dashboards/", "").replace(".yaml", ""))
+      // Sort alphabetically case-insensitive
+      .sort((a, b) => a.localeCompare(b, undefined, { sensitivity: "base" }));
```

Comment:
It should use [`ListCatalogEntries`](https://github.com/rilldata/rill-developer/blob/933c7bcca003061cff93e3614a939bd702b33d60/proto/rill/runtime/v1/api.proto#L90), which takes a `?type=OBJECT_TYPE_METRICS_VIEW` parameter to only return dashboards.

This way, it doesn't list dashboards that fail to reconcile, and will also continue to work if we add support for new project folder structures